xyz.ljoonal.neospeeps.yml

```diff
@@ -0,0 +1,50 @@
+app-id: xyz.ljoonal.neospeeps
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm13
+command: neos_peeps
+finish-args:
+  # Needs to talk to the network:
+  - --share=network
+  # Wayland access
+  - --socket=wayland
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm13/bin
+  env: 
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER: clang
+    CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS: -C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: clang,
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS: -C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold
+modules:
+  - name: neos_peeps
+    buildsystem: simple
+    only-arches:
+      - x86_64
```

Comment:
this doesn't do what you think it does. 

Also why do you want to limit to x86_64. If you don't build aarch64 surely NO ONE will test it.

Reply:
https://buildbot.flathub.org/#/builders/7/builds/8201 seemed to indicate that building on aarch64 failed, so I thought it'd make more sense to not keep trying to build while it's known to be broken? And if it doesn't do what I think it does, browsing trough the other PRs, should I instead add a `flathub.json` file which defines it then?

Reply:
that's what the documentation says. But why do you need "clang" as linker?

Reply:
but again it's disappointing to ignore aarch64

Reply:
> But why do you need "clang" as linker?

I basically just followed https://github.com/flathub/org.freedesktop.Sdk.Extension.rust-stable whilst trying to figure out how the whole flatpak building process worked, and was already familiar with mold so thought I'd just go ahead and keep using it everywhere.
Is there an issue with using it?

> but again it's disappointing to ignore aarch64

I mean, true I guess, but I'm already struggling to manage distributing this to x64 only, and I've yet to have a single person say that they'd use an aarch64 build.

Reply:
so basically you don't know. I'd just remove clang altogether. It's not needed to build anything with Rust, that will remove any variable.